packages/form-js-editor/src/features/dragging/Dragging.js

```diff
@@ -78,8 +78,8 @@ export class Dragging {
     let targetParentId;
 
     if (formFieldNode) {
-      formField = this._formFieldRegistry.get(formFieldNode.dataset.id);
-      columns = (formField.layout || {}).columns;
+      formField = this._formFieldRegistry.get(formFieldNode.dataset.id) || { layout: {} };
+      columns = formField.layout.columns;
```

Comment:
@Skaiir this bug seems to be only happening on the Desktop modeler on Windows, I couldn't reproduce it locally but this is the issue if you follow the stack trace

Could you give it a try and reproduce it on your machine? (It should be ok to merge this anyway)